frontend/src/components/Main/UserInfo.js

```diff
@@ -0,0 +1,84 @@
+import React, {Component} from "react";
+import {Button, Form} from "react-bootstrap";
+import RunAnalysisService from "../Service/RunAnalysisService";
+import ModelFilter from "../Filters/ModelFilter";
+import OptionsFilter from "../Filters/OptionsFilter";
+import DataFilter from "../Filters/DataFilter";
+
+import {toJS} from "mobx";
+
+import {inject, observer} from "mobx-react";
+
+@inject("DataStore")
+@observer
+class UserInfo extends Component {
+    constructor(props) {
+        super(props);
+        this.runAnalysisService = new RunAnalysisService();
+        this.modelFilter = new ModelFilter();
+        this.optionFilter = new OptionsFilter();
+        this.dataFilter = new DataFilter();
+    }
+
+    handleSubmit = event => {
```

Comment:
the `handleSubmit` method should be an action in the datastore. The filters should also be attached at the datastore